mapbox_tilesets/errors.py

```diff
@@ -24,4 +24,8 @@ class TilesetNameError(TilesetsError):
     """
 
     def __init__(self, tileset_id):
-        self.message = f"{tileset_id} is not a valid Tileset ID"
+        self.tileset_id = tileset_id
+        self.message = "Invalid Tileset ID"
```

Comment:
I'm not opposed to using the `__str__` method, but could we have just added the name into the error message here?

Reply:
@dnomadb do you mean that you would prefer to put the tileset id into `self.message` so `__str__` can reference only `self.message`?

Reply:
> Or are you asking why we need __str__ at all

Yeah, I am curious on this part: usually I would just use the class name + message to communicate what needs to be passed. Again not blocking so will approve!

Reply:
@dnomadb did you see the `before` output in the PR description? Happy to implement this in a different way if you have a suggestion, but the current way doesn't work.

Reply:
Oh, got it! 

I was thinking we were doing:
```python
tileset_id = "invalid"

raise TilesetNameError(f"{tileset_id} is not a valid Tileset ID")
---------------------------------------------------------------------------
TilesetNameError                          Traceback (most recent call last)
<ipython-input-5-21fcd5287e0c> in <module>
----> 1 raise TilesetNameError(f"{tileset_id} is not a valid Tileset ID")

TilesetNameError: invalid is not a valid Tileset ID
```
But I now see that this formatting is in the error class itself, which doesn't seem to work. Sorry for missing that @dianeschulze !

Reply:
No worries, appreciate the check!